src/server.c

```diff
@@ -5921,6 +5921,7 @@ int main(int argc, char **argv) {
             }
         }
     } else {
+        loadSentinelConfigFromQueue();
```

Comment:
maybe there's an advantage to do this right after `loadServerConfig` rather than here?
i suppose most of the code in between these lines is only using non-sentinel-specific configs.
but for instance if we exit on unrecognized config here that's happening after daemonization.

Reply:
thanks @oranagra , you are right, it is good to load it when the loadServerConfig is done.